src/codebook/defaultSettings.js

```diff
@@ -10,6 +10,7 @@ const defaultSettings = {
   nBins: 100,
   levelSplit: 5, //cutpoint for # of levels to use levelPlot() renderer
   controlVisibility: 'visible',
+  chartVisibility: 'visible',
```

Comment:
This default isn't being effected.

Reply:
Fixed typo in `setDefaults.js`, and bit the bullet to change the default to `minimized`